tests/local.rs

```diff
@@ -10,6 +9,11 @@ use datachannel::{
     RtcConfig, RtcDataChannel, RtcPeerConnection, SessionDescription,
 };
 
+#[cfg(feature = "log")]
+use log::*;
+#[cfg(feature = "tracing")]
+use tracing::*;
```

Comment:
How about:

```rs
use crate::logger;

// ...
logger::info!("DataChannel PING: Open");
// etc ...
```
?

Reply:
And the same `logger::info` etc could be used everywhere

Reply:
I can pull in these macros if you want, but then they will be available to all crates that use datachannel and that may not be the best thing...

Reply:
As long as `mod logger` is not `pub` and you don't use [`macro_export`](https://doc.rust-lang.org/reference/macros-by-example.html#path-based-scope) the macros won't be available to crates downstream.

I think that just using `use crate::logger;` is enough to bring the `logger` module in scope, and from there you can access its macros with `logger::info!` (without making the macros available downstream).